bodhi/tests/server/services/test_updates.py

```diff
@@ -285,13 +285,13 @@ def test_new_rpm_update_from_tag(self, *args):
         # We don't want the new update to obsolete the existing one.
         self.db.delete(Update.query.one())
 
-        update = self.get_update(builds=None, from_tag='f17-updates-candidate')
+        update = self.get_update(builds=None, from_tag='f17-build-side-7777')
```

Comment:
These changes will probably conflict with https://github.com/fedora-infra/bodhi/pull/3481, it will be nice to merge this first